### PR TITLE
docs(platform): fdp link examples

### PR DESCRIPTION
**Pull request by droshev**
fdp link examples

### CODE REVIEW

File: libs/docs/platform/link/examples/platform-link-example.component.html [hunk 1]
```diff
@@ -1,65 +1,13 @@
...
-            href="http://www.google.com"
-            [truncate]="true"
-            [title]="'this is a long text link. so getting truncated.'"
-        >
-            truncated text that very long
-        </fdp-link>
-    </p>
-    link.
-</div>
+<h4 fd-title>Example of a linkn</h4>
```
Comment (InnaAtanasova) [inline libs/docs/platform/link/examples/platform-link-example.component.html:1]: Typo in `linkn`

File: libs/docs/platform/link/examples/platform-link-example.component.html [hunk 2]
```diff
@@ -1,65 +1,13 @@
...
+<fdp-link href="http://www.google.com" [title]="'Extra info as tooltip text'"> google search </fdp-link>
+<br /><br />
+<h4 fd-title>Example of a link with an icon</h4>
```
Comment (InnaAtanasova) [inline libs/docs/platform/link/examples/platform-link-example.component.html:4]: Please remove this `fd-title`, use just `<h4>`.
The reason: the title of the documentation, the one outside of the example section, doesn't have `fd-title` and now visually they look similar

File: libs/docs/platform/link/examples/platform-link-example.component.html (hunk 1 above)
Comment (droshev) [reply libs/docs/platform/link/examples/platform-link-example.component.html]: fixed

File: libs/docs/platform/link/examples/platform-link-example.component.html (hunk 2 above)
Comment (droshev) [reply libs/docs/platform/link/examples/platform-link-example.component.html]: done

**Review by InnaAtanasova — APPROVED** [review]

**Pull request merged**